sql/src/main/java/io/crate/expression/scalar/DateTruncFunction.java

```diff
@@ -61,9 +61,9 @@
         .immutableMap();
 
     public static void register(ScalarFunctionModule module) {
-        List<DataType> supportedTimestampTypes = ImmutableList.of(
-            DataTypes.TIMESTAMPZ, DataTypes.LONG, DataTypes.STRING);
-        for (DataType dataType : supportedTimestampTypes) {
+        List<DataType<?>> supportedTimestampTypes = ImmutableList.of(
+            DataTypes.TIMESTAMP, DataTypes.TIMESTAMPZ, DataTypes.LONG, DataTypes.STRING);
+        for (DataType<?> dataType : supportedTimestampTypes) {
             module.register(new DateTruncFunction(info(DataTypes.STRING, dataType)));
             // time zone aware variant
             module.register(new DateTruncFunction(info(DataTypes.STRING, DataTypes.STRING, dataType)));
```

Comment:
do we also register date_trunc for `timestamp without timezone`  for the `time zone aware variant` ?